lms/static/js/commerce/views/receipt_view.js

```diff
@@ -39,7 +39,7 @@ var edx = edx || {};
                 courseKey: this.courseKey
             });
 
-            this.$el.html(_.template(templateHtml)(context));
+            edx.HtmlUtils.setHtml(this.$el, edx.HtmlUtils.template(templateHtml)(context));
```

Comment:
Nit: if you have time, it might be cleaner to pass ``edx.HtmlUtils`` to the anonymous outer function, so you don't have to duplicate the namespace everywhere. This is very minor so only do it if you're changing the code anyway.